.ci/generate_operators_doc.py

```diff
@@ -0,0 +1,152 @@
+from ansys.dpf import core as dpf
+from ansys.dpf.core.dpf_operator import available_operator_names
+from ansys.dpf.core.core import load_library
+import argparse
+from jinja2 import Template
+from pathlib import Path
+
+def initialize_server(ansys_path=None):
+    server = dpf.start_local_server(ansys_path=ansys_path)
+    print(f"Ansys Path: {server.ansys_path}")
+    print(f"Server Info: {server.info}")
+    print(f"Server Context: {server.context}")
+    print(f"Server Config: {server.config}")
+    print(f"Server version: {dpf.global_server().version}")
+    print("Loading Composites Plugin")
+    load_library(
+        Path(server.ansys_path) / "dpf" / "plugins" / "dpf_composites" / "composite_operators.dll"
+    )
+    print("Loading Acoustics Plugin")
+    load_library(Path(server.ansys_path) / "Acoustics" / "SAS" / "ads" / "dpf_sound.dll")
```

Comment:
Hi @ansMHanmer, these two plugins will not be available in the PyDPF-Core pipelines. Could you make them optional?